src/SelectPanel/SelectPanel.tsx

```diff
@@ -129,9 +124,19 @@ export function SelectPanel({
 
   const inputRef = React.useRef<HTMLInputElement>(null)
   const focusTrapSettings = {
-    initialFocusRef: inputRef
+    initialFocusRef: inputRef,
+    disabled: textInputProps?.disabled
   }
 
+  const extendedTextInputProps: Partial<TextInputProps> = useMemo(() => {
+    return {
+      mx: 2,
+      my: 2,
+      contrast: true,
+      ...textInputProps
+    }
+  }, [textInputProps])
```

Comment:
Transferring comment from https://github.com/lukewar/components/commit/b0cacf73b28717bdd246aaa7f45ec93a19ecf514#r52246102

Two suggestions here:
1. We should spread the `textInputProps` _after_ the defaults, so that all of them can be overridden if needed
2. We should memoize this value with `useMemo` to avoid churn on the `FilteredActionList` props

Reply:
Thanks for the feedback. Addressed.